lib/dynflow/action/v2/with_sub_plans.rb

```diff
@@ -173,6 +175,7 @@ def concurrency_limit_capacity
     def cancel!(force = false)
       # Count the not-yet-planned tasks as cancelled
       output[:cancelled_count] = total_count - output[:planned_count]
+      output[:cancelled_timestamp] = DateTime.now.new_offset(0).to_s # time in UTC for comparison with UTC times in the database
```

Comment:
In theory, `cancel!` can be called multiple times, in which case it should keep the oldest timestamp